### PR TITLE
Generic file viewer (download area) page styling

### PR DESCRIPTION
**Pull request by awikkerink**
Related: https://github.com/Brightspace/activity-viewer/pull/84
https://rally1.rallydev.com/#/42651186647d/detail/userstory/56146901337

### CODE REVIEW

File: src/plugins/generic/generic.scss [hunk 1]
```diff
@@ -88,55 +96,51 @@
...
 		button {
-			margin-top: 0.25rem;
+			margin-top: 10px;
+			margin-bottom: 10px;
 			width: 90%;
 		}
 
-		.generic-header-icon-container {
+		.generic-download-area {
 			flex-direction: column-reverse;
```
Comment (jstefaniuk-d2l) [inline src/plugins/generic/generic.scss]: Do we want to `column-reverse` this in mobile?

**Comment by jstefaniuk-d2l** [comment]
Other than my one comment, LGTM

File: src/plugins/generic/generic.scss (hunk 1 above)
Comment (awikkerink) [inline src/plugins/generic/generic.scss]: Yes. This is intentional

Comment (jstefaniuk-d2l) [inline src/plugins/generic/generic.scss]: So in mobile, the size text and download button switch places?

Comment (awikkerink) [inline src/plugins/generic/generic.scss]: This is intentional. https://rally1.rallydev.com/#/42651186647d/detail/userstory/56146901337 The file size and download button are switch in mobile

**Comment by jstefaniuk-d2l** [comment]
:+1:

**Comment by awikkerink** [comment]
@cgalvind2l Looks good? To test,

`npm run build-sample`
`cd sample`
`python -mSimpleHTTPServer`
open localhost:8000

**Comment by cgalvind2l** [comment]
LGTM

**Pull request merged**